### DISCUSSION TITLE
No lib/forge-std folder after executing forge init --force

### DISCUSSION THREAD

**Discussion by Lakshya-Garg07** [Cyfrin/foundry-full-course-cu#3058] [Q&A] [answered]
After executing the forge init --force, there is no lib folder being created. The folders that are getting created are script,src and test. Here is the terminal output after executing force init --force command.
```
Warning: Target directory is not empty, but `--force` was specified
Initializing /home/lakshya/Blockchain/Foundry/Foundry-Simple-Storage...
Error: failed to commit (code=Some(128), stdout="", stderr="Author identity unknown\n\n*** Please tell me who you are.\n\nRun\n\n  git config --global user.email \"you@example.com\"\n  git config --global user.name \"Your Name\"\n\nto set your account's default identity.\nOmit --global to set the identity only in this repository.\n\nfatal: empty ident name (for <lakshya@MY-ASUS.>) not allowed")
```

**Comment by EngrPips** [comment]
Hello @Lakshya-Garg07, can you initiate a new Foundry project in a new empty folder?

**Reply by Lakshya-Garg07** [reply]
Hi @EngrPips, I tried that too, but still no lib folder was created.

**Reply by EngrPips** [reply, marked as answer]
Run the below command in your terminal and let's see if the `lib` folder will get populated.

```js
forge install foundry-rs/forge-std --no-commit
```

**Reply by Lakshya-Garg07** [reply]
Oh yeah, that worked. Thank you so much :) .

**Reply by EngrPips** [reply]
Awesome, I am glad that worked. Keep crushing it.
